models/contacts.js

```diff
@@ -1,19 +1,115 @@
-// const fs = require('fs/promises')
+import * as path from 'path';
+import { readFile, writeFile } from 'fs/promises';
+import { nanoid } from 'nanoid';
 
-const listContacts = async () => {}
+const contactsPath = path.resolve('./models', 'contacts.json');
 
-const getContactById = async (contactId) => {}
+const readContactsFile = async () => {
```

Comment:
this is fine but personally i would name it just `getContacts`. It's not importat from where u get contacts. Today it is file, tomorrow it can be database.

Reply:
important is what you get in response and method name should reflect that